src/rabbitmanager.js

```diff
@@ -111,7 +111,7 @@ class RabbitManager {
 
   async deleteUserPermissions(user, vhost='/') {
     vhost = encodeURIComponent(vhost);
-    await this.request(`permissions/${vhost}/${user}`, {method: 'DELETE'});
+    await this.request(`permissions/${vhost}/${user}`, {method: 'delete'});
```

Comment:
I made that change for consistency, since all of the other REST verbs are spelt in lowercase. Either way, I have not seen uppercase or lowercase impact performance or functionality.